pkg/tenant/api/v1/client_test.go

```diff
@@ -300,171 +300,147 @@ func TestTenantDelete(t *testing.T) {
-
 	}))
 	defer ts.Close()
 
-	// Creates a client to execute tests against the test server
 	client, err := api.New(ts.URL)
-	require.NoError(t, err)
+	require.NoError(t, err, "could not create api client")
 
-	out, err := client.AppCreate(context.Background(), &api.App{})
-	require.NoError(t, err)
-	require.Equal(t, fixture.ID, out.ID)
-	require.Equal(t, fixture.AppName, out.AppName)
+	req := &api.PageQuery{}
+
+	out, err := client.TenantMemberList(context.TODO(), "tenant01", req)
+	require.NoError(t, err, "could not execute api request")
+	require.Equal(t, fixture, out, "unexpected result occurred")
 }
 
-func TestAppDetail(t *testing.T) {
-	fixture := &api.App{
-		ID:      "001",
-		AppName: "username01",
+func TestTenantMemberCreate(t *testing.T) {
+	fixture := &api.TenantMember{
+		TenantID:   "01",
+		MemberID:   "02",
+		MemberName: "Luke Hamilton",
+		MemberRole: "Admin",
 	}
 
-	// Creates a test server
 	ts := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-		require.Equal(t, http.MethodGet, r.Method)
-		require.Equal(t, "/v1/apps/:id", r.URL.Path)
+		require.Equal(t, http.MethodPost, r.Method)
+		require.Equal(t, "/v1/tenant/tenant01/members", r.URL.Path)
+
+		in := &api.TenantMember{}
+		err := json.NewDecoder(r.Body).Decode(in)
+		require.NoError(t, err, "could not decode request")
 
 		w.Header().Add("Content-Type", "application/json; charset=utf-8")
-		w.WriteHeader(http.StatusOK)
+		w.WriteHeader(http.StatusNoContent)
 		json.NewEncoder(w).Encode(fixture)
-
 	}))
 	defer ts.Close()
 
-	// Creates a client to execute tests against the test server
 	client, err := api.New(ts.URL)
-	require.NoError(t, err)
+	require.NoError(t, err, "could not create api client")
 
-	req := &api.App{
-		ID:      "001",
-		AppName: "username01",
+	req := &api.TenantMember{
+		TenantID:   "01",
+		MemberID:   "02",
+		MemberName: "Luke Hamilton",
+		MemberRole: "Admin",
 	}
 
-	out, err := client.AppDetail(context.Background(), req.ID)
-	require.NoError(t, err)
-	require.Equal(t, fixture.ID, out.ID)
-	require.Equal(t, fixture.AppName, out.AppName)
+	err = client.TenantMemberCreate(context.TODO(), "tenant01", req)
+	require.NoError(t, err, "could not execute api request")
 }
 
-func TestTopicList(t *testing.T) {
-	fixture := &api.TopicPage{}
+func TestMemberList(t *testing.T) {
+	fixture := &api.MemberPage{
+		Members: []*api.Member{
+			{
+				ID:         "002",
+				MemberName: "Ryan Moore",
+				MemberRole: "Admin",
+			},
+		},
+		PrevPageToken: "2121",
+		NextPageToken: "2123",
+	}
 	// Creates a test server
 	ts := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 		require.Equal(t, http.MethodGet, r.Method)
-		require.Equal(t, "/v1/topics", r.URL.Path)
+		require.Equal(t, "/v1/members", r.URL.Path)
 
-		w.Header().Add("Content-Type", "application/json; charset=utf-8")
-		w.WriteHeader(http.StatusOK)
-		json.NewEncoder(w).Encode(fixture)
+		params := url.Values{}
```

Comment:
This change to how params are set in the test was made so that the URL will no longer need to be explicitly stated.